### DISCUSSION TITLE
Feature implement, how to select cascade data

### DISCUSSION THREAD

**Discussion by qjcjerry** [nocodb/nocodb#489] [Features]
Feature implement, how to select cascade data? like show in following pic.
![image](https://user-images.githubusercontent.com/6712585/130392880-0f475c69-70c6-4ba4-ba00-3736e1fff081.png)

**Comment by o1lab** [comment]
From which tool is the screenshot from ?

**Reply by qjcjerry** [reply]
snipaste

**Reply by o1lab** [reply]
No I meant, on which tool (like SQL tool?) - did you do take that snipaste ? :)

**Reply by qjcjerry** [reply]
it's a ant design front end react library, you can see that in link below.
https://ant.design/components/cascader/
